avm-transpiler/bootstrap.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eu
+
+cd $(dirname "$0")
+
+CMD=${1:-}
+
+if [ -n "$CMD" ]; then
+  if [ "$CMD" = "clean" ]; then
+    git clean -fdx
```

Comment:
could through in a cargo clean in here too

Reply:
done